addons/customisable-control-buttons/addon.json

```diff
@@ -0,0 +1,162 @@
+{
+    "name": "Customisable control buttons",
+    "description": "Allows you to customise the play, pause (if you have the addon enabled) and stop buttons.",
+    "credits": [
+      {
+        "name": "Callumjt",
+        "link": "https://scratch.mit.edu/users/callumjt/"
+      }
+    ],
+    "versionAdded": "1.31.0",
+    "userstyles": [
+      {
+        "url": "style.css",
+        "matches": ["projects"]
+      },
+      {
+        "url": "color.css",
+        "matches": ["projects"],
+        "if": {
+          "settings": { "buttonsBackground": true }
+        }
+      },
+      {
+        "url": "hover.css",
+        "matches": ["projects"],
+        "if": {
+          "settings": { "hover": true, "buttonsBackground": false }
+        }
+      },
+      {
+        "url": "hover2.css",
+        "matches": ["projects"],
+        "if": {
+          "settings": { "hover": true, "buttonsBackground": true }
+        }
+      }
+    ],
+    "tags": ["projectPlayer"],
+    "enabledByDefault": false,
+    "dynamicEnable": true,
+    "dynamicDisable": true,
+    "settings": [
+      {
+        "name": "Button width (32-48px)",
+        "id": "buttonWidth",
+        "type": "integer",
+        "min": 32,
+        "default": 32,
+        "max": 48
+      },
+      {
+        "name": "Button gap (0-5px)",
+        "id": "buttonGap",
+        "type": "integer",
+        "min": 0,
+        "default": 3,
+        "max": 5
+      },
+      {
+        "name": "Border radius (0-100px)",
+        "id": "borderRadius",
+        "type": "integer",
+        "min": 0,
+        "default": 4,
+        "max": 100
+      },
+      {
+        "name": "Remove hover (must refresh to apply)",
+        "id": "hover",
+        "type": "boolean",
+        "default": false
+      },
+      {
+        "name": "Buttons background (must refresh to apply)",

```

Comment:
The "must refresh to apply" can probable be fixed with `"updateUserstylesOnSettingsChange": true` in the root of the manifest

Reply:
ah ok thank you